File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cs
using System;

public class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        string ra = a.ToString() + b.ToString();
        int rb = 2 * a * b;
        if(int.Parse(ra) >= rb){
            answer = int.Parse(ra);
        }
        else {answer = rb;}
        return answer;
    }
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) {
    int answer = 0;
    int s = stoi(to_string(a) + to_string(b));
    if(s >= 2*a*b)
        answer = s;
        
    else  answer = 2*a*b;
    return answer;
}